### PR TITLE
Evict chunks

### PR DESCRIPTION
**Pull request by HereThereBeDragons**
Issue #3122 

Code changes from @mharvey-jt + integration test

### CODE REVIEW

File: test/src/705-evict/main [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+
+
+  echo "   ...Success"
+  echo "   Unmount repo"
+  private_unmount
+  echo "*** FINISHED Testing evict_file for file: /$filename"
+  echo ""
+}
+
+evict_and_readd() {
```
Comment (jblomer) [inline test/src/705-evict/main:44]:
```suggestion
evict_and_read() {
```

**Review by jblomer — APPROVED** [review]
Very nice.  Is `evict_and readd` a typo or intended?

**Comment by jblomer** [comment]
@cernvm-bot cloudtest

File: test/src/705-evict/main (hunk 1 above)
Comment (HereThereBeDragons) [reply test/src/705-evict/main:44]: i wanted to say re-add .. but i guess evict_and_read could also work :-D

**Pull request merged**